e2e/tests/snapshots/golden_tests__testdata__create_ingest_temp_table.sql.snap

```diff
@@ -9,18 +9,20 @@ expression: query_result
  (3,"aVeryLongMultiCaseMetricWhich""might""BeTruncatedIfIt'sTOOOOOO_3",t)
 (3 rows)
 
-                                               runtests                                               
-------------------------------------------------------------------------------------------------------
+psql:/testdata/create_ingest_temp_table.sql:41: NOTICE:  identifier "ThisIsLong_aVeryLongMultiCaseMetricWhich"might"BeTruncatedIfIt'sTOOOOOO_3" will be truncated to "ThisIsLong_aVeryLongMultiCaseMetricWhich"might"BeTruncatedIfIt'"
```

Comment:
I am concerned about this truncation.

Reply:
Let me explain my concern in more detail.

Promscale handles long metric names by creating a unique truncated version of the metric name. This happens in `get_or_create_metric_table_name`. As we see in the test, `aVeryLongMultiCaseMetricWhich"might"BeTruncatedIfIt''sTOOOOOOOOOOOOOOOLONGZOMG!` gets truncated to `ThisIsLong_aVeryLongMultiCaseMetricWhich"might"BeTruncatedIfIt'sTOOOOOO_3`. This is still uniquely identifiable (thanks to `_3` suffix).

If we now prepend something to that unique identifier, and the suffix becomes truncated, it's no longer possible to uniquely identify the table. This is not just inconvenient, it could result in the same temporary table being reused for different metrics, which would result in totally unexpected data corruption.

Reply:
I see what you mean. It's a valid concern. However, in this case we create temporary tables which are bound to a session (a connection) and there we provide uniqueness guarantees by generating unique prefix (this is connector code). I hope this makes it clear?

Reply:
So to summarize during insert in one session all metrics will have unique temporary tables.